client/public/index.html

```diff
@@ -16,6 +16,10 @@
       type="text/css"
       href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Material+Icons"
```

Comment:
can probably remove Roboto and keep Material+Icons, if you are not using Roboto anymore